### PR TITLE
feat(env-checker): [part-1] copy env checker code to CLI and remove useless part

### PR DESCRIPTION
**Pull request by a1exwang**
- Move environment checker code to CLI
- Rename env checker adapter/logger/telemetry in CLI
- Update framework code accordingly (copy scripts, package.json)
- Remove `execSudo` because we don't need it anymore and CLI does not have this package.

The plan is that there should be another 3 PRs following this.
- Implement adapters in CLI
- Implement the high level env checker in preview command
- Implement feature flags with Config in CLI

task item: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10338893

### CODE REVIEW

**Comment by meifans** [comment]
LGTM

**Review by meifans — APPROVED** [review]

File: packages/cli/src/cmds/preview/depsChecker/cliLogger.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as os from "os";
+import cliLogger from "../../../commonlib/log";
+import { LogLevel } from "@microsoft/teamsfx-api";
+import { IDepsLogger } from "./checker";
+
+export class VSCodeLogger implements IDepsLogger {
```
Comment (kuojianlu) [inline packages/cli/src/cmds/preview/depsChecker/cliLogger.ts:9]: Should this name be modified?

**Review by Alive-Fish — APPROVED** [review]
Approved for `cli/package.json`.

File: packages/cli/src/cmds/preview/depsChecker/cliLogger.ts (hunk 1 above)
Comment (a1exwang) [reply packages/cli/src/cmds/preview/depsChecker/cliLogger.ts]: Good catch. Renamed all "vscode" to "cli"

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**